wallets/provider-phantom/src/legacy/index.ts

```diff
@@ -24,7 +28,15 @@ export const config = {
 };
 
 export const getInstance = phantom_instance;
-export const connect: Connect = getSolanaAccounts;
+const connect: Connect = async ({ instance, meta }) => {
+  const solanaInstance = instance.get(Networks.SOLANA);
+  const result = await getSolanaAccounts({
```

Comment:
It seems that you are trying to keep support of legacy providers but you didn't add `connect` and `sign` for `evm` namespace. If it is not possible, at least it should be documented.

Reply:
Good point. I added a comment on code to explain we will not add new functionality to legacy anymore.

Reply:
But you have added `ETHEREUM` and `POLYGON` to supported chains.

Reply:
It will not be needed eventually when adapter removed its dependency to `wallentInfo.supportedChains`. I mentioned `In Future` section.